miso-web/src/main/webapp/pages/editSample.jsp

```diff
@@ -329,6 +337,89 @@
   </c:if>
 </div>
 </form:form>
+<c:if test="${sample.id == 0}">
+</div>
+<div id="tab-2">
+  <div class="breadcrumbs">
+    <ul>
+      <li>
+        <a href="/">Home</a>
+      </li>
+      <c:if test="${not empty sample.project}">
+      <li>
+        <div class="breadcrumbsbubbleInfo">
+          <div class="trigger">
+            <a href='<c:url value="/miso/project/${sample.project.id}"/>'>${sample.project.name}</a>
+          </div>
+          <div class="breadcrumbspopup">
+              ${sample.project.alias}
+          </div>
+        </div>
+      </li>
+      </c:if>
+    </ul>
+  </div>
+  <h1>Create Samples
+    <button id="bulkSampleButton" onClick="submitBulkSamples();" class="fg-button ui-state-default ui-corner-all">
+      Save
+    </button>
+  </h1>
+  <br/>
+  <br/>
+
+
+  <div id="HOTbulkForm" data-detailed-sample="${detailedSample}">
+    <div class="floatleft">
+	    <div>Project:<select id="projectSelect"></select></div>
+	    <div id="subpSelectOptions"></div>
+	    <div id="classOptions"></div>
+	    <div id="saveSuccesses"  class="parsley-success hidden">
+	      <p id="successMessages"></p>
+	    </div>
+	    <div id="saveErrors" class="bs-callout bs-callout-warning hidden">
+	      <h2>Oh snap!</h2>
+	      <p>The following rows failed to save:</p>
+	      <p id="errorMessages"></p>
+	    </div>
+	    <div>
+	      <button id="makeTable" class="disabled" disabled="true" onclick="Sample.hot.checkForExistingHOT();">Make Table</button>
+	      <button id="rerender" class="disabled" disabled="true" onclick="Sample.hot.hotTable.render();">Readjust Table View</button>
+	    </div>
+	  </div>
+	  <div class="floatleft rightfloatleft">
+      <p>To fill all columns below with the value of your selected cell, <b>double-click</b> the square in the bottom right of your selected cell.
+      To fill a variable number of columns with the value of your selected cell, 
+      <b>click</b> the square in the bottom right of your filled-in selected cell and <b>drag</b> up or down. All selected columns will be filled in.
+    </div>
+	  
+	  <div style="clear:both;"></div>
+    <div>
+      <c:choose>
+      <c:when test="${detailedSample}">
+        <button id="saveDetailed" class="disabled" disabled="true" onclick="Sample.hot.saveDetailedData();">Save</button>
+        <div id="genAliasesButton"></div>
+      </c:when>
+      <c:otherwise>
+        <button id="savePlain" onclick="Sample.hot.savePlainData();">Save Bulk</button>
+      </c:otherwise>
+      </c:choose>
+    </div>
+    <div id="hotContainer"></div>
+  
+    <script type="text/javascript">
+      Sample.hot.dropdownRef = JSON.parse('${referenceDataJson}');
+      Sample.hot.selectedProjectId = parseInt('${sample.project.id}') || null;
+      Sample.hot.detailedSample = JSON.parse(document.getElementById('HOTbulkForm').dataset.detailedSample);
+      if (Boolean("${detailedSample}")) {
```

Comment:
`<c:if>` inside the <script> tag? That'll break it good. But I store the boolean one line up, so I"ll use that instead.